source/mongodump.txt

```diff
@@ -302,8 +302,33 @@ value overrides the read preference specified in the URI string.
 Data Exclusion
 ~~~~~~~~~~~~~~
 
-:binary:`~bin.mongodump` excludes the content of the ``local`` database
-in its output.
+If you specify both the :option:`--db <mongodump --db>` and the
+:option:`--collection <mongodump --collection>` options, ``mongodump``
+dumps the specified collection.
+
+If you only specify the :option:`--db <mongodump --db>` option,
+``mongodump`` dumps that database with the following exclusions:
+
+- If the value of ``--db`` is ``admin``, ``mongodump`` excludes the
+  ``system.keys`` collection. 
+- If the value of ``--db`` is ``config``, ``mongodump`` dumps all
+  collections in that database. 
+  
+  If you don't explicitly specify the ``config`` database as the value
+  for the  ``--db`` option or if you omit the ``--db`` option,
+  ``mongodump`` only includes the following collections in the output
+  for the ``config`` database: 
+
+  - ``chunks``
+  - ``collections``
+  - ``databases``
+  - ``settings``
+  - ``shards``
+  - ``tags``
+  - ``version``
```

Comment:
Explicitly specifying --db config/admin is documented above, so that's implicit in the list below

```suggestion
  If you omit the ``--db`` option:
  - ``mongodump`` excludes the ``local`` database.
  - ``mongodump`` only includes the following collections in the output
    for the ``config`` database: 

    - ``chunks``
    - ``collections``
    - ``databases``
    - ``settings``
    - ``shards``
    - ``tags``
    - ``version``
    
  - If you're connected to an Atlas Proxy, ``mongodump`` excludes the ``admin`` 
    database.
```